serialization/class_definition.go

```diff
@@ -37,6 +37,14 @@ func NewClassDefinition(factoryID int32, classID int32, version int32) *ClassDef
 	}
 }
 
+func NewClassDefinitionDefault(factoryID int32, classID int32) *ClassDefinition {
```

Comment:
Let's remove this function.

Reply:
Removed, [5dca40b](https://github.com/hazelcast/hazelcast-go-client/pull/758/commits/5dca40bd3b2c69bfe4310bb3c293c71d358b2116).